requirements.txt

```diff
@@ -28,6 +28,7 @@ uszipcode
 pypostalcode
 watchtower # for cloudwatch logging
 sphinx # for code documentation
+rsa==4.3
```

Comment:
You should get this change if you merge in the main branch -- Tim merged it earlier.

Reply:
I rebased on the recent version of the master and reverted my changes